apps/condo/domains/subscription/schema/ServiceSubscription.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Generated by `createschema subscription.ServiceSubscription 'type:Text; isTrial:Checkbox; organization:Relationship:Organization:CASCADE; startAt:DateTimeUtc; finishAt:DateTimeUtc;'`
+ */
+
+const { Select, Checkbox, DateTimeUtc, Integer, Decimal } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/subscription/access/ServiceSubscription')
+const { ORGANIZATION_OWNED_FIELD } = require('../../../schema/_common')
+const { ServiceSubscription: ServiceSubscriptionAPI } = require('../utils/serverSchema')
+const get = require('lodash/get')
+const { OVERLAPPING_ERROR } = require('../constants/errors')
+
+
+const ServiceSubscription = new GQLListSchema('ServiceSubscription', {
+    schemaDoc: 'Availability time period of service features for client organization. Can be trial or payed.',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        type: {
+            schemaDoc: 'System, from where subscription was created (our or external)',
+            type: Select,
+            options: 'default,sbbol',
+            isRequired: true,
+        },
+
+        isTrial: {
```

Comment:
It’s better to use date for that

Reply:
end trial period date

Reply:
You said, that default and SBBOL subscriptions can be trial.
Subscription of any kind has start and finish time.
For what specific case You think it is better to duplicate this fields with `startTrial` and `finishTrial`?

Reply:
Discussed today, current implementation is ok.